app/models/metric/container_storage.rb

```diff
@@ -0,0 +1,17 @@
+module Metric::ContainerStorage
+  def self.fill_allocated_container_storage(obj)
+    volumes_or_claims = if obj.kind_of? ContainerGroup
```

Comment:
IIRC @nimrodshn added pod => pvc relation too, so it might be simpler to use it here.